include/ofi_mr.h

```diff
@@ -100,8 +101,9 @@ static inline uint64_t ofi_mr_get_prov_mode(uint32_t version,
 }
 
 
-/* Single lock used by all memory monitors and MR caches. */
+/* Locks used by all memory monitors and MR caches. */
```

Comment:
Would you please elaborate here? Specifically what do these locks protect?

Reply:
Done